src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Player_goal.json

```diff
@@ -2128,7 +2128,7 @@
     ],
     "returnTag": {
       "comment": "True if the player can afford the purchase.",
-      "condition": "",
+      "condition": "target.gold >= args[0] ? result==true",
```

Comment:
I think this translation comes from the code, but is actually difficult (impossible?) to infer from the comment itself.
If this is the case, I suggest to leave the translation empty.